package.json

```
@@ -25,36 +25,37 @@
     "build:swagger": "tsx example/generate-open-api-schema.ts",
     "build:client": "tsx example/generate-client.ts && yarn prettier example/example.client.ts --write",
     "build:license": "tsx tools/license.ts",
-    "build:logo": "tsx tools/startup-logo.mts",
+    "build:logo": "tsx tools/startup-logo.ts",
     "test": "yarn test:types && yarn test:unit && yarn test:badge",
-    "test:int": "yarn --cwd tests/integration && vitest run tests/integration",
+    "test:cjs": "yarn --cwd tests/cjs && vitest run -r tests/cjs",
+    "test:esm": "yarn --cwd tests/esm && vitest run -r tests/esm",
     "test:952": "yarn --cwd tests/issue952 && yarn --cwd tests/issue952 test",
-    "test:esm": "yarn --cwd tests/esm && vitest run tests/esm",
     "test:compat": "yarn --cwd tests/compat && yarn --cwd tests/compat test",
     "test:u": "yarn test:unit -u && yarn test:badge",
     "test:types": "tsc --noEmit",
-    "test:unit": "vitest run tests/unit tests/system",
+    "test:unit": "vitest run --coverage tests/unit tests/system",
     "test:badge": "make-coverage-badge --output-path coverage.svg",
     "lint": "eslint src example tests tools && yarn prettier *.md --check",
     "mdfix": "prettier *.md --write",
-    "cleanup": "rm -rf tests/**/node_modules coverage tests/**/coverage",
+    "cleanup": "rm -rf tests/**/node_modules coverage",
     "precommit": "yarn lint && yarn test && yarn build && git add example/example.* LICENSE coverage.svg",
     "prepublishOnly": "yarn lint && yarn test && yarn build",
     "postversion": "git push && git push --tags",
     "install_hooks": "husky install"
   },
-  "main": "dist/index.js",
+  "type": "module",
+  "main": "dist/index.cjs",
```

According to examples I saw, this should now also point to ESM version, not CJS

```suggestion
  "main": "dist/index.js",
```

---

decided to keep CJS for weird compatibility cases.
all findings tell that it really does not matter if `exports` is specified